server/blob/blob-objectstorage/src/main/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAO.java

```diff
@@ -43,6 +45,8 @@
 import reactor.core.publisher.Mono;
 
 public class ObjectStorageBlobsDAO implements BlobStore {
```

Comment:
This is not a DAO right? And DAO do not perform retries.

Is it the time to rename it `ObjectStorageBlobStore` ?

Reply:
> This is not a DAO right? And DAO do not perform retries.

Why? I thought a DAO was designed to get the job done.

Reply:
I agree with @chibenwa , a DAO should only have the logic to access the data.
The retry logic, if any, should be at an upper level.
BTW, I don't understand why we need to retry a `get`, could you explain why we need that?

Reply:
IMO we can rename this class ^^

Reply:
The Swift container fails on some `read` on heavy loads.